utils/funcexp/func_if.cpp

```diff
@@ -53,22 +53,22 @@ bool boolVal(SPTP& parm, Row& row, const string& timeZone)
             case CalpontSystemCatalog::TEXT:
             case CalpontSystemCatalog::VARCHAR:
                 ret = (atoi((char*)(parm->data()->getStrVal(timeZone).c_str())) != 0);
-
+                break;
```

Comment:
Did I get it right that we have at least 3 years old bug here?

Reply:
um... Yes